Gemfile.jruby-2.3.lock.release

```diff
@@ -322,7 +322,7 @@ GEM
       mail (~> 2.6.3)
       mime-types (= 2.6.2)
       stud (~> 0.0.22)
-    logstash-input-jdbc (4.2.4)
+    logstash-input-jdbc (4.3.0)
```

Comment:
This fixes behavior from here: https://github.com/logstash-plugins/logstash-input-jdbc/issues/147